settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,55 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: cloud
+              options:
+                  cloud:
+                      type: Value
+                      help: The cloud which the OpenStack modules will operate against
+                      required: yes
+            - title: image
+              options:
+                  image:
+                      type: Value
+                      help: An image id or name, on OpenStack cloud to provision the instance with
```

Comment:
We discussed adding the "glance image-list" command to the `--image` help text so users can know which flavors are available for them.